scalereg/scale_templates/reg6/reg_kiosk.html

```diff
@@ -29,7 +29,7 @@
     <table border="0" width="100%">
     <tr>
     <td align="left">
-    <p>Registration</p>
+    <p>SCALE Registration (Station {{ agent }})</p>
```

Comment:
Pull event name from settings?

Reply:
I left out "SCALE" from the line. Parameterizing the event name in various templates would be a PR all on its own.